react-native/ios/Keybase/AppDelegate.swift

```diff
@@ -43,6 +43,30 @@ class AppDelegate: UIResponder {
   
 }
 
+class KeyListener: UIViewController {
+  override func canBecomeFirstResponder() -> Bool {
+    return true
+  }
+
+  var bridge: RCTBridge!
+
+  override var keyCommands: [UIKeyCommand]? {
+    return [
+      UIKeyCommand(input: "b", modifierFlags: .Command, action: "goBackInTime:"),
+      UIKeyCommand(input: "n", modifierFlags: .Command, action: "goForwardInTime:")
```

Comment:
i have all sorts of problems with command+n, seems to disconnect my chrome and reset my sessions and whatnot